cypress/integration/prospects/profile.js

```diff
@@ -17,4 +17,46 @@ describe('Prospects profile info', () => {
       cy.go('back')
     }
   })
+  it('Should have permissions when logged in as user', () => {
+
+    cy.login('normal@example.com', 'password')
+    cy.visit('http://localhost:3000/prospects/list')
+
+    // open each prospect profile by clicking on the prospect card
+    for (let index = 1; index < 10; ++index) {
+      // open the prospect profile
+      cy.get('div[id="prospect-' + index + '"').click()
+
+      // Open the profile menu and check if it is visible
+      cy.get('#profileAction').click();
+      cy.get('#delete-button').should('be.visible');
+
+      // Click the first and only item in the main menu,
+      // open the modal window and confirm the action
+      cy.get('ul li').should('have.length', 1);
+      cy.get('#delete-button').click();
+      cy.get('#modalConfirm').should('be.visible');
+      cy.get('#confirmActionMessage')
+        .should('be.visible')
+        .contains('Are you sure you want to delete Prospect ' + index + '?');
+      cy.get('#confirmAction').click();
+
+      // Check the URL and navigate back to the profile
+      cy.url().should('eq', 'http://localhost:3000/prospects/list');
+      cy.go('back');
+
+      // Open the profile menu again, check the menu,
+      // click the first item in the menu, cancel the action,
+      // and verify that the modal window is gone
+      cy.get('#profileAction').click();
+      cy.get('#delete-button').should('be.visible');
+      cy.get('#delete-button').click();
+      cy.get('#modalConfirm').should('be.visible');
+      cy.get('#cancelAction').click();
+      cy.get('#modalConfirm').should('not.exist');
```

Comment:
In the near future when you will be "actually deleting" the prospect, these tests will not pass any longer, so, let's make two separated `for` loops that go from 0 to 5. The first loop will check the "confirm" button and the second loop will check the "cancel" button.